camera.py

```diff
@@ -39,30 +44,45 @@ def open(self, filename=1):
 class CameraVM016(Camera):
     def __init__(self):
         super().__init__()
-        self.color_conversion_code = cv.COLOR_BAYER_GB2RGB
+
```

Comment:
Set the `use_isp` property here as an initialization paramter instead of modifying the member variable from outside the class.
```
class CameraVM016(Camera):
    def __init__(self, isp: bool = False):
        super().__init__()
```